inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/ClickHouseLoadNode.java

```diff
@@ -100,7 +121,7 @@ public String genTableName() {
 
     @Override
     public String getPrimaryKey() {
-        return super.getPrimaryKey();
+        return primaryKey;
     }
 
     @Override

```

Comment:
MayBe this is unnecessary.Clickhouse sink currently is implement by jdbc connector.But [jdbc table connector  sink](https://nightlies.apache.org/flink/flink-docs-master/api/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableSink.html) don't support applyPartition currently. So get partitionFiled is useless.

Reply:
partitionFiled is useless, the method for obtaining partitionFiled is not modified here, but the getPrimaryKey() method is used